### ISSUE TITLE
Input issues remain after Mjolnir update

### ISSUE THREAD

**Issue by Shoegzer** [yuzu-emu/yuzu#4945]
After the latest update to yuzu-master which includes PR https://github.com/yuzu-emu/yuzu/pull/4866, rumble effects are still missing from Pinball FX 3 and Star Wars Pinball, which this PR was reported to have resolved.

Also, nothing appears under the "Profile" drop-down input menu, and the Save | New | Delete buttons don't do anything when clicked.

Specs:

OS: Linux Mint 20, kernel 5.4
CPU: Intel i7-7820HK
GPU: Nvidia 1070M, driver 455.38
Compiler: gcc 9.3.0
Build: abda36636245c416a75774165d2a5b49610952fc (master-mainline-636-3743-ge3672ee10)
Log: [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/5561766/yuzu_log.txt)

**Comment by german77** [comment]
Are you sure that your build is correct? Please test on mainline and not a custom build. I will confirm in a few hours

**Comment by Shoegzer** [comment]
Thanks @german77, yes it's correct.  I compiled as I always have done against yuzu-mainline, no changes were made to make it custom.

**Comment by german77** [comment]
I will have to test on mainline. I have the mjolnir changes in my dev build and confirmed that everything worked as described. Can't test at the moment until I get home.

**Comment by Shoegzer** [comment]
No worries @german77 , and actually I figured out what was wrong - I was building with ` -DCMAKE_BUILD_TYPE=Release` which must have recently been changed to default and is no longer needed as a CMAKE flag.  Removing that compiles the latest version, which is actually `master-mainline-636-3785-gbc7c76409` rather than the older build I reported above.  Thanks for working with me on this though.  Closing.
